hack/update-bazel.sh

```diff
@@ -20,7 +20,11 @@ set -o pipefail
 export KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 source "${KUBE_ROOT}/hack/lib/init.sh"
 
-go get gopkg.in/mikedanese/gazel.v16/gazel
+# Remove generated files prior to running gazel.
+# TODO(spxtr): Remove this line once Bazel is the only way to build.
+rm -f "${KUBE_ROOT}/pkg/generated/openapi/zz_generated.openapi.go"
```

Comment:
In my PR I went the other way - force it to be generated before running gazel.  Otherwise some symbols are missing for other genfiles